core/settings.py

```diff
@@ -420,6 +425,15 @@ def output_list(self, value):
         # We're gonna trust the user not to output to a location without write permission
         self._output_list = pathlib.Path(value).resolve()
 
+    @property
+    def unavaiable_list(self):
+        return self._unavaiable_list
+
+    @unavaiable_list.setter
+    def unavaiable_list(self, value):
+        # We're gonna trust the user not to output to a location without write permission
+        self._unavaiable_list = pathlib.Path(value).resolve()    
```

Comment:
Please follow the example of archive.setter, which is stricter than output_list.setter